packages/plugin-calendar/src/calendar.js

```diff
@@ -0,0 +1,33 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+
+const Calendar = SparkPlugin.extend({
+  namespace: `Calendar`,
+
+  /**
+   * Retrieves a collection of meetings based on the request parameters
+   * @param {Object} options
+   * @param {Date} options.fromDate the start of the time range
+   * @param {Date} options.toDate the end of the time range
+   * @returns {Promise} Resolves with an array of meetings
+   */
+  get(options) {
+    options = options || {};
+    return this.spark.device.register().then(() => {
```

Comment:
Note that in it's current form, this promise chain will resolve with the result of `spark.device.register()` because you're not returning `this.spark.request()`